authentication/views.py

```diff
@@ -0,0 +1,67 @@
+from django.shortcuts import redirect, render, HttpResponse
+from django.contrib.auth.models import User
+from django.contrib.sites.shortcuts import get_current_site
+from django.urls import reverse
+from django.contrib import messages
+from django.conf import settings
+from django.contrib.auth import login, logout
+from django.shortcuts import render, redirect
+from django.contrib.auth import authenticate, login
+
+#User Login
+def login_page(request):
+    if not request.user.is_authenticated:
+        if request.method == 'POST':
+            email = request.POST['email']
+            password = request.POST['psw']
+
+            if User.objects.filter(email=email).exists():
+                user = User.objects.get(email=email)
+                if user.check_password(password):
+                    login(request, user)
+                    return redirect("#")
```

Comment:
What does the `#` in `redirect("#")` mean? Is this a typo?

Reply:
After login, this part will return to the user's homepage/profile. whereas we don't create this page, I just write this "#".

Reply:
Okay, thanks for the explanation! 😀